src/lib/sse.ts

```diff
@@ -1,25 +1,47 @@
-let connections: Set<ReadableStreamDefaultController> = new Set();
+let connections: Map<string, Set<ReadableStreamDefaultController>> = new Map();
```

Comment:
I don't think that multiple connections from the same client should be possible.

Reply:
Multiple tabs

Reply:
Oh, Ok